content/docs/concepts/csidriver/features/powerstore.md

```diff
@@ -419,6 +419,10 @@ allowedTopologies:
 
 This example matches all nodes where the driver has a connection to PowerStore with an IP of `127.0.0.1` via FibreChannel. Similar examples can be found in mentioned folder for NFS, iSCSI and NVMe.
 
+**Note** : 
```

Comment:
Haven't we seen the same behavior for other drivers? We should update it for all drivers and even in older versions. Next question/bug could be for another platform/driver.

Reply:
Updated in older versions also, as we are releasing soon so skipped for v3